coverage_control2/Polygon outer_boundary
coverage_control2/Polygon[] holes